politica.csl

```diff
@@ -0,0 +1,282 @@
+            <text variable="container-title" font-style="italic" suffix=" "/>
+            <group>
+              <choose>
+                <if is-numeric="volume">
+                  <choose>
+                    <if match="all" variable="issue" is-numeric="volume">
+                      <group>
+                        <number variable="volume"/>
+                        <number prefix=" (" suffix=")" variable="issue"/>
+                      </group>
+                    </if>
+                    <else>
+                      <number variable="volume"/>
+                    </else>
+                  </choose>
+                </if>
+                <else>
+                  <text variable="volume"/>
+                </else>
+              </choose>
+              <text macro="pages" prefix=": " suffix="."/>
+            </group>
+          </group>
+        </else-if>
+        <else-if type="thesis">
+          <text macro="year-date" prefix=" (" suffix=")."/>
+          <text macro="title" prefix=" " suffix="."/>
+          <group prefix=" " delimiter=", ">
+            <text macro="edition"/>
+            <text macro="editor" suffix="."/>
+            <text variable="genre"/>
+            <text macro="publisher"/>
+          </group>
+        </else-if>
+        <else-if type="bill book graphic legal_case legislation motion_picture report song" match="any">
+          <group delimiter=" ">
+            <text macro="year-date" prefix=" (" suffix=")."/>
+            <text macro="title" suffix="." font-style="italic"/>
+            <text macro="edition"/>
+            <text macro="editor" suffix="."/>
+            <text macro="publisher"/>
+          </group>
+        </else-if>
+        <else-if type="chapter paper-conference" match="any">
+          <group delimiter=" ">
+            <text macro="year-date" prefix=" (" suffix=")."/>
+            <text macro="title" suffix="."/>
+            <group delimiter=", " suffix=",">
+              <group delimiter=" ">
+                <text variable="page" prefix="pp. "/>
+                <text macro="editor" prefix="i " suffix="."/>
+                <text variable="container-title" font-style="italic"/>
+              </group>
+              <choose>
+                <if is-numeric="volume">
+                  <number variable="volume" suffix=". årgang"/>
+                </if>
+                <else>
+                  <text variable="volume"/>
+                </else>
+              </choose>
+              <text variable="collection-title"/>
+            </group>
+            <text macro="publisher"/>
+          </group>
+        </else-if>
+        <else>
+          <group suffix=" " prefix=" " delimiter=" ">
+            <text macro="year-date" prefix="(" suffix=")."/>
+            <text macro="title" prefix="" suffix="."/>
+            <text macro="editor" suffix="."/>
+          </group>
+          <group prefix=" " delimiter=" ">
+            <text variable="container-title" suffix=","/>
```

Comment:
Move the comma to the group delimiters to avoid duplication of punctuation when item data is missing